x-pack/plugins/uptime/public/components/overview/alerts/settings_message_expression_popover.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiExpression, EuiPopover } from '@elastic/eui';
+import { Link } from 'react-router-dom';
+import React, { useState } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { SETTINGS_ROUTE } from '../../../../common/constants';
+
+interface SettingsMessageExpressionPopoverProps {
+  'aria-label': string;
+  description: string;
+  id: string;
+  setAlertFlyoutVisible: (value: boolean) => void;
+  value: string;
+}
+
+export const SettingsMessageExpressionPopover: React.FC<SettingsMessageExpressionPopoverProps> = ({
+  'aria-label': ariaLabel,
+  description,
+  setAlertFlyoutVisible,
+  value,
+  id,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  return (
+    <EuiPopover
+      id={id}
+      anchorPosition="downLeft"
+      button={
+        <EuiExpression
+          aria-label={ariaLabel}
+          color="secondary"
+          description={description}
+          isActive={isOpen}
+          onClick={() => setIsOpen(!isOpen)}
+          value={value}
+        />
+      }
+      isOpen={isOpen}
+      closePopover={() => setIsOpen(false)}
+    >
+      <FormattedMessage
+        id="xpack.uptime.alerts.tls.settingsPageNav.text"
+        defaultMessage="You can edit these thresholds on the {settingsPageLink}."
```

Comment:
Should it be less verbose?
instead of saying 'You can edit these thresholds on the settings page.'
`You can edit this threshold in Settings.` though i am not a native english speaker
so you probably know better.

Reply:
The copy comes from [the spec](https://github.com/elastic/uptime/issues/162), so I think we can just leave it for now.